docker_explorer/explorer.py

```diff
@@ -204,14 +204,14 @@ def GetContainersJson(self, only_running=False):
 
       mount_points = container_object.GetMountpoints()
       if mount_points:
+        container_json['mount_points'] = []
         for source, destination in mount_points:
           mountpoint_dict = collections.OrderedDict()
           mountpoint_dict['source'] = os.path.join(
               self.docker_directory, source)
           mountpoint_dict['destination'] = os.path.join(
               os.path.sep, destination)
           container_json['mount_points'].append(mountpoint_dict)
```

Comment:
I think the problem is the extra s, it is setdefault not setdefaults 
this should work `container-json.setdefault('mount_points', []).append(mountpoint_dict)`

Reply:
also the `container_json['mount_points'].append(mountpoint_dict)` is not needed if you add the setdefault line

Reply:
damn I've been really stupid here. Fixed